app/services/katello/registration_manager.rb

```diff
@@ -297,6 +297,7 @@ def remove_host_artifacts(host, clear_content_facet = true)
           host.content_facet.applicable_errata = []
           host.content_facet.uuid = nil
           host.content_facet.save!
+          host.content_facet.calculate_and_import_applicability
```

Comment:
calling in the applicability expert @ianballou 

I was originally looking at just zeroing out the counts. But I called `calculate_and_import_applicability` here because it seems to also handle Deb, Rpm, and Module streams and I wanted to make sure nothing was messed up there:
```rb
::Katello::Applicability::ApplicableContentHelper.new(self, ::Katello::Deb, bound_repos).calculate_and_import
        ::Katello::Applicability::ApplicableContentHelper.new(self, ::Katello::Rpm, bound_repos).calculate_and_import
        ::Katello::Applicability::ApplicableContentHelper.new(self, ::Katello::Erratum, bound_repos).calculate_and_import
        ::Katello::Applicability::ApplicableContentHelper.new(self, ::Katello::ModuleStream, bound_repos).calculate_and_import
```

Is that the right decision? Or do you think it will be a performance issue to do this when unregistering hosts?